api/net/Areas/Editor/Controllers/WorkOrderController.cs

```diff
@@ -100,7 +100,7 @@ public IActionResult Find()
     {
         var uri = new Uri(this.Request.GetDisplayUrl());
         var query = Microsoft.AspNetCore.WebUtilities.QueryHelpers.ParseQuery(uri.Query);
-        var result = _workOrderService.FindDistinctWorkOrders(new WorkOrderFilter(query), this._serializerOptions);
+        var result = _workOrderService.FindDistinctWorkOrders(new WorkOrderFilter(query));
```

Comment:
This was implemented incorrectly.  The serialization options should be applied through dependency injection, like it is every where else.